src/jdk.crypto.cryptoki/share/classes/sun/security/pkcs11/wrapper/Functions.java

```diff
@@ -740,12 +783,14 @@ private static void addMGF(long id, String name) {
         addMech(CKM_SHA512_T_HMAC_GENERAL,      "CKM_SHA512_T_HMAC_GENERAL");
         addMech(CKM_SHA512_T_KEY_DERIVATION,    "CKM_SHA512_T_KEY_DERIVATION");
 
-        addMech(CKM_SECURID_KEY_GEN,            "CKM_SECURID_KEY_GEN");
-        addMech(CKM_SECURID,                    "CKM_SECURID");
-        addMech(CKM_HOTP_KEY_GEN,               "CKM_HOTP_KEY_GEN");
-        addMech(CKM_HOTP,                       "CKM_HOTP");
-        addMech(CKM_ACTI,                       "CKM_ACTI");
-        addMech(CKM_ACTI_KEY_GEN,               "CKM_ACTI_KEY_GEN");
+        addMech(CKM_SHA3_256_RSA_PKCS,          "CKM_SHA3_256_RSA_PKCS");
+        addMech(CKM_SHA3_384_RSA_PKCS,          "CKM_SHA3_384_RSA_PKCS");
+        addMech(CKM_SHA3_512_RSA_PKCS,          "CKM_SHA3_512_RSA_PKCS");
+        addMech(CKM_SHA3_256_RSA_PKCS_PSS,      "CKM_SHA3_256_RSA_PKCS_PSS");
+        addMech(CKM_SHA3_384_RSA_PKCS_PSS,      "CKM_SHA3_384_RSA_PKCS_PSS");
+        addMech(CKM_SHA3_512_RSA_PKCS_PSS,      "CKM_SHA3_512_RSA_PKCS_PSS");
+        addMech(CKM_SHA3_224_RSA_PKCS,          "CKM_SHA3_224_RSA_PKCS");
+        addMech(CKM_SHA3_224_RSA_PKCS_PSS,      "CKM_SHA3_224_RSA_PKCS_PSS");
```

Comment:
It appears that you're arranging the addMech(with CKM_xxx) based on the mechanism values. How about the code from #773 to #793, move it up?

Reply:
Yes, I will move it up and add a comment about overall ordering is based on PKCS11Constants class.